README.md

```diff
@@ -158,9 +166,599 @@ search.start();
+- ✅ attribute: The facet to display _required_
+- ✅ operator: How to apply facets, "AND" or "OR"
+- ✅ limit: How many facet values to retrieve.
+- ✅ showMore: Whether to display a button that expands the number of items.
+- ✅ showMoreLimit: The maximum number of displayed items. Does not work when showMoreLimit > limit.
+- ❌ searchable: Whether to add a search input to let the user search for more facet values. Not supported by MeiliSearch. If you'd like to see it implemented [please vote](https://roadmap.meilisearch.com/c/64-search-for-facet-values?utm_medium=social&utm_source=portal_share).
+- ❌ searchablePlaceholder: The value of the search input’s placeholder, not supported, see `searchable`.
+- ❌ searchableIsAlwaysActive: When false, disables the facet search input, not supported, see `searchable`.
+- ❌ searchableEscapeFacetValues: When true, escapes the facet values, not supported, see `searchable`.
+- ❌ sortBy: Not supported natively but can be implemented manually using `transformItems` options.
+- ✅ transformItems: A function to transform the items passed to the templates.
+- ✅ templates: The templates to use for the widget.
+- ✅ cssClasses: The CSS classes to override.
+
+The following example will create a UI component with the a list of genres on which you will be able to facet.
+
+```js
+instantsearch.widgets.refinementList({
+  container: '#refinement-list',
+  attribute: 'genres',
+})
+```
+
+### ❌ HierarchicalMenu
+
+[Hierarchical menu references](https://www.algolia.com/doc/api-reference/widgets/hierarchical-menu/js/)
+
+The `hierarchicalMenu` widget is used to create a navigation based on a hierarchy of facet attributes. It is commonly used for categories with subcategories.
+
+No compatibility because MeiliSearch does not support hierarchical facets.
+
+If you'd like get nested facets/hierarchical facets implemented, please vote for it in the [roadmap](https://roadmap.meilisearch.com/c/97-nested-facets?utm_medium=social&utm_source=portal_share).
+
+### ✅ RangeSlider
+
+[Range slider references](https://www.algolia.com/doc/api-reference/widgets/range-slider/js/)
+
+The `rangeSlider` widget provides a user-friendly way to filter the results, based on a single numeric range.
+
+- ✅ container: The CSS Selector or HTMLElement to insert the refinements. _required_
+- ✅ attribute: The name of the attribute in the document. _required_.
+- ✅ min: The minimum value for the input. _required_
+- ✅ max: The maximum value for the input. _required_
+- ❌ precision: The number of digits after the decimal point to use. Not compatible as only integers work with `rangeSlider`.
+- ✅ step: The number of steps between each handle move.
+- ✅ pips: Whether to show slider pips (ruler marks).
+- ✅ tooltips: Whether to show tooltips. The default tooltips show the raw value.
+- ✅ cssClasses: The CSS classes to override.
+
+⚠️ The component is compatible but requires the following workaround:https://www.algolia.com/doc/api-reference/widgets/js/
```

Comment:
```suggestion
⚠️ The component is compatible but requires the following workaround: https://www.algolia.com/doc/api-reference/widgets/js/
```

Reply:
I don't get which workaround we need to do

Reply:
Do you mean the next step following? It looks like the workaround is in the link you put after the sentence